azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template): Fix loot tables for a few rare npcs 2

## Changes Proposed:
-  Add a 100% chance to drop a green item to the rares listed in the issue. This is done by replacing reference_loot_template items 24037, 24039, 24041, 24056 which have a 1% chance for the rares with a single new list with 100% chance.


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5451



## SOURCE:



## Tests Performed:

- Tested in game, spawned rares and killed them before and after the SQL change.
- 


## How to Test the Changes:


| ID | Creature |
| -- | -- |
| 763 | Lost One Chieftain |
| 2605 | Zalas Witherbark |
| 2604 | Molok the Crusher |
| 14234 | Hayoc |
| 14226 | Kaskk |
| 2744 | Shadowforge Commander |
 
1. Spawn any of the npcs from the above list (`.npc add <ID>`)
2. Kill the `npc .damage 100000`
3. Loot
4. Verify that green item is drop with 100% chance


## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
